roles/maas_region_controller/tasks/main.yaml

```diff
@@ -1,5 +1,10 @@
 ---
 # Playbook to generate a MAAS region
+- name: Verify MAAS Version supported on OS
```

Comment:
should this also run in the rack controller role or does this check only need to run once to assert the version is in fact valid? If the latter, `run_once` could be used to only run this on a single region controller (for reference: https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html)

Reply:
You're right, I forgot to add this to the rack controller too!
The way I have it currently, this will run on every host (and only fail the hosts that have incompatible `maas_version` given their `distro_release`)
If we would rather the whole playbook fair we can change that.